hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/service/launcher/TestServiceInterruptHandling.java

```diff
@@ -93,7 +94,7 @@ public void testBlockingShutdownTimeouts() throws Throwable {
     InterruptEscalator escalator = new InterruptEscalator(launcher, 500);
     // call the interrupt operation directly
     try {
-      escalator.interrupted(new IrqHandler.InterruptData("INT", 3));
+      escalator.interrupted(new IrqHandler.InterruptData("SIGINT", 3));
```

Comment:
we should move to LambdaTestUtils.intercept() here instead of the try/catch clause
```
assertExceptionDetails(EXIT_INTERRUPTED, "", 
  intercept(ExitUtil.ExitException.class, "", () ->
    escalator.interrupted(new IrqHandler.InterruptData("SIGINT", 3))));
    
```